src/components/navBar/Navbar.js

```diff
@@ -37,9 +36,9 @@ const Navbar = ({ data, year }) => (
   </>
 );
 
-Navbar.propTypes = {
-  data: PropTypes.string.isRequired,
-  year: PropTypes.number,
-};
+// Navbar.propTypes = {
+//   data: PropTypes.string.isRequired,
+//   year: PropTypes.number,
+// };
```

Comment:
- [ ] Kindly remove this commented code in order to have a clean codebase, thanks. 🙏